README.md

```diff
@@ -10,15 +10,15 @@ Steps to run a local copy of SpartaHackAPI
     ```
       pip3 install virtualenvwrapper
     ```  
-   And add the following three lines to your shell startup file (.bashrc, .profile, etc.)  
+   And add the following three lines to your shell startup file (.zshrc, .bashrc, .profile, etc.)  
     ```
-      export VIRTUALENVWRAPPER_PYTHON=/usr/bin/python3
+      export VIRTUALENVWRAPPER_PYTHON=/usr/local/bin/python3
```

Comment:
/usr/bin/python3 is for ubuntu I guess. You can add another entry for MacOS. That would be a better way because ubuntu has python installed in /bin and not /local/bin

Reply:
yeah, I'll look again where it's at and change some things there.